packages/components/addon/components/hds/pagination/nav/index.js

```diff
@@ -75,6 +83,46 @@ export default class HdsPaginationNavIndexComponent extends Component {
     return this.currentPage === this.totalPages;
   }
 
+  elliptize({ pages, current }) {
+    const array = Array.from(pages);
+    const limit = 7; // limt # of page numbers shown at a time
```

Comment:
sorry, there is a bug in the algorithm (the bug is already in the code I sent you as demo, so my bad)
if you change the limit to 9 you will see that it doesn't work as expected
the reason is in this line:
```
result = [].concat(sliceStart.shift(), ellipsis, sliceCurr, ellipsis, sliceEnd.pop());
```
which assumes that the first block and the last block are of 1 element which is not true if the limit is different (eg. `9`). 

It's not a blocker per se, and it's a "bug" only if one changes the limit from `7` (which btw is something we should consider as an option, seems more easy to use/grasp, maybe check with @cveigt what he thinks)